src/saltext/vmware/modules/cluster.py

```diff
@@ -83,40 +83,48 @@ def create(name, datacenter, service_instance=None):
     return {name: True}
 
 
-def get_(name, datacenter, service_instance=None):
+def get_(cluster_name, datacenter_name, service_instance=None):
     """
     Get the properties of a cluster.
 
-    Supported proxies: esxcluster
-
-    name
+    cluster_name
         The cluster name
 
-    datacenter
+    datacenter_name
         The datacenter name to which the cluster belongs
 
     .. code-block:: bash
```

Comment:
Should we have a docstring for service_instance?

Reply:
will add